src/dali/plugin/input/rest/coinbase_pro.py

```diff
@@ -123,6 +123,16 @@ def __init__(
         self.__session: Session = requests.Session()
         self.__logger: logging.Logger = create_logger(f"{self.__COINBASE_PRO}/{self.account_holder}")
 
+        hash_state = hashlib.sha256()
```

Comment:
To avoid the security warning about SHA256, try: 
```suggestion
         hash_state = hashlib.sha3_256()
```